.travis.yml

```diff
@@ -73,6 +73,7 @@ after_success:
   else
    ./gradlew coveralls;
   fi
+- python <(curl -s https://raw.githubusercontent.com/TouK/sputnik-ci/master/sputnik-ci.py)
```

Comment:
This makes me a little nervous (downloading and executing some python script from github)...